MyConfig.h

```diff
@@ -1551,6 +1551,33 @@
  */
 //#define MY_MQTT_CLIENT_KEY
 
+/**
+ * @def MY_MQTT_ETH_INIT_DELAY
+ * @brief Set a delay for Ethernet initialisation.
+ *
+ * This define is useful if you want to change default Ethernet chip initialisation
+ * delay time to other value. By default, it is 1000ms
+ * Example: @code #define MY_MQTT_ETH_INIT_DELAY 1 @endcode
+ *
+ * @endcode
+ */
+#ifndef MY_MQTT_ETH_INIT_DELAY
+#define MY_MQTT_ETH_INIT_DELAY 1000
+#endif
+/**
+ * @def MY_MQTT_ETH_CLIENT_CONNECTION_TIMEOUT
+ * @brief Set a MQTT broker socket connection timeout time.
+ *
+ * This define is useful if you want to change default MQTT TCP/IP broker
+ * connection timeout. By default, it is 1000ms.
+ * Example: @code #define MY_MQTT_ETH_CLIENT_CONNECTION_TIMEOUT 1000 @endcode
+ *
+ * @endcode
```

Comment:
One @endcode too many (the endcode at line 1573 is enough). Just remove this line.

Reply:
done ;)